pom.xml

```diff
@@ -66,6 +66,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-resteasy-reactive-jackson</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
```

Comment:
If not necessary, I prefer to use the non-reactive client for now.

Reply:
Applied